exporter/otelarrowexporter/config.go

```diff
@@ -65,32 +67,26 @@ type ArrowSettings struct {
 	// Note that `Zstd` applies to gRPC, not Arrow compression.
 	PayloadCompression configcompression.Type `mapstructure:"payload_compression"`
 
-	// Disabled prevents using OTel Arrow streams.  The exporter
+	// Disabled prevents using OTel-Arrow streams.  The exporter
 	// falls back to standard OTLP.
 	Disabled bool `mapstructure:"disabled"`
 
 	// DisableDowngrade prevents this exporter from fallback back
 	// to standard OTLP.  If the Arrow service is unavailable, it
 	// will retry and/or fail.
 	DisableDowngrade bool `mapstructure:"disable_downgrade"`
+
+	// Prioritizer is a policy name for how load is distributed
+	// across streams.
+	Prioritizer arrow.PrioritizerName `mapstructure:"prioritizer"`
```

Comment:
This config option is not described in the README. It's not a big issue since the component is in development stability.